Week03/decorators_furkan_eryilmaz.py

```diff
@@ -0,0 +1,18 @@
+import time
+import sys
+
+def performance(function_):
+    def wrapper(*args, **kwargs):
+        start_time = time.time()
+        result = function_(*args, **kwargs)
+        end_time = time.time()
+
+        if not hasattr(performance, 'counter'):
+            performance.counter = 0
+            performance.total_time = 0
+            performance.total_mem = 0
+
+        performance.counter += 1
+        performance.total_time += end_time - start_time
+        performance.total_mem += sys.getsizeof(result)
```

Comment:
You just get the size of the return value, I want you to check the total memory that this function uses.